C++/Logger/TypeIs.h

```diff
@@ -2,19 +2,19 @@
 #define _TYPE_IS_H_
 
 template <typename T>
-struct type_is
+struct TypeIs
```

Comment:
I know we stated the preferred style for the project but I am inclined to agree with you for template metafunctions.